src/components/media-loader.js

```diff
@@ -457,7 +458,7 @@ AFRAME.registerComponent("media-loader", {
           { once: true }
         );
         this.el.setAttribute("floaty-object", { reduceAngularFloat: true, releaseGravity: -1 });
-        let batch = !disableBatching;
+        let batch = !disableBatching && forceImageBatching;
```

Comment:
Maybe defer to the `batchManagerSystem.batchingEnabled` instead of `disableBatching` (but keep the check for `forceImageBatching`)

Reply:
Yeah was going to do that but then became slightly afraid of system initialization bugs. Will probably do that when enabling batching again though